game.c

```diff
@@ -108,13 +108,7 @@ void setup_level() {
         mobs[i].stacks = false;
     }
 
-    player = &mobs[MAX_MOBS-1];
-    player->x = player->y = 1;
-    player->behavior = KeyboardInput;
-    player->display = ICON_HUMAN;
-    player->active = true;
-
-    for (int i=0; i < 100; i++) {
+    for (int i=0; i < MAX_MOBS; i++) {
```

Comment:
I guess the actual mob initialization will normally happen during level creation so this loop will disappear soon anyway.